srcs/Request.cpp

```diff
@@ -141,6 +141,48 @@ Request::set_m_error_code(int error_code)
 /*********************************  Util  *************************************/
 /*============================================================================*/
 
+std::string
+Request::getMethod()
+{
+	if (this->m_method == GET)
+		return ("GET");
+	else if (this->m_method == HEAD)
+		return ("HEAD");
+	else if (this->m_method == POST)
+		return ("POST");
+	else if (this->m_method == PUT)
+		return ("PUT");
+	else if (this->m_method == DELETE)
+		return ("DELETE");
+	else if (this->m_method == OPTIONS)
+		return ("OPTIONS");
+	else if (this->m_method == TRACE)
+		return ("TRACE");
+	return ("DEFAULT");
+}
+
+std::string
+Request::getContentLength()
+{
+	std::map<std::string, std::string>::const_iterator it;
+
+	it = this->m_headers.find("Content-Length");
+	if (it == this->m_headers.end())
+		return ("");
+	return (it->second);
+}
+
+std::string
+Request::getContentType()
+{
+	std::map<std::string, std::string>::const_iterator it;
+
+	it = this->m_headers.find("Content-Type");
+	if (it == this->m_headers.end())
+		return ("text/html");
```

Comment:
```suggestion
		return ("text/plain");
```

> 특정 서브타입이 없는 텍스트 문서들에 대해서는 text/plain가 사용되어야 합니다. 특정 혹은 알려진 서브타입이 없는 이진 문서에 대해서는 유사하게, application/octet-stream이 사용되어야 합니다.

text/html은 default로 나중에 다른 처리가 힘들것같아서 공식문서 정의대로 text/plain가 좋을것같아요!

- https://developer.mozilla.org/ko/docs/Web/HTTP/Basics_of_HTTP/MIME_types